baseplate/context/memcache/lib.py

```diff
@@ -44,13 +44,16 @@ class Flags(object):
 
 
 def decompress_and_load(key, serialized, flags):
-    """Deserialization method compatible with dump_and_compress().
+    """Modern deserialization method.
```

Comment:
I'm lacking some context, but what makes this Modern? What is the un-modern equivalent? Something with memcache protocol/clients?